src/core/public/core_app/status/components/metric_tiles.tsx

```diff
@@ -7,24 +7,99 @@
  */
 
 import React, { FunctionComponent } from 'react';
-import { EuiFlexGrid, EuiFlexItem, EuiCard } from '@elastic/eui';
-import { formatNumber, Metric } from '../lib';
+import { EuiFlexGrid, EuiFlexItem, EuiCard, EuiStat } from '@elastic/eui';
+import { DataType, formatNumber, Metric } from '../lib';
 
 /*
- * Displays a metric with the correct format.
+ * Displays metadata for a metric.
  */
-export const MetricTile: FunctionComponent<{ metric: Metric }> = ({ metric }) => {
-  const { name } = metric;
+export const MetricCardFooter: FunctionComponent<{
+  testSubjectName: string;
+  title: string;
+  description: string;
+}> = ({ testSubjectName, title, description }) => {
   return (
-    <EuiCard
-      data-test-subj={`serverMetric-${formatMetricId(metric)}`}
-      layout="horizontal"
-      title={formatMetric(metric)}
-      description={name}
+    <EuiStat
+      data-test-subj={testSubjectName}
+      title={title}
+      titleSize="xxs"
+      description={description}
+      reverse
     />
   );
 };
 
+/*
+ * Displays a metric with the correct format.
+ */
+export const MetricTile: FunctionComponent<{ metric: Metric }> = ({ metric }) => {
+  const { name } = metric;
+  if (name === 'Delay') {
```

Comment:
nit: I'd probably go with a `switch` here for readability, falling back to a `default`

Reply:
I think I would even create sub-components for each of the cases and just have this one behaves as a dispatcher. 

```tsx
export const MetricTile: FunctionComponent<{ metric: Metric }> = ({ metric }) => {
  switch(metric.name) {
     case 'Delay': return <DelayMetricTile ... />
     case 'Load': return <LoadMetricTile ... />
     // ....
   }

}
```

That's not mandatory though, but probably more readable and isolated.